shm/include/fruit.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "cargo.hpp"
+
+class Fruit : public Cargo {
+public:
+    Fruit(const std::string& name, size_t amount, double basePrice);
+    Fruit(const std::string& name, size_t amount, double basePrice, size_t timeToRotten);
+    ~Fruit() override {}
+
+    double getPrice() const override {
+        return getBasePrice() * static_cast<double>((getTimeToRotten()) / (getTimeToRotten() + timeElapsed_));
+    }
```

Comment:
Only simply getter in .hpp. Please move to cpp

Reply:
done